test/unit/activeselection.js

```diff
@@ -28,7 +28,7 @@
   QUnit.module('fabric.ActiveSelection', {
     afterEach: function() {
       canvas.clear();
-      canvas.backgroundColor = fabric.Canvas.prototype.backgroundColor;
+      canvas.backgroundColor = canvas.getDefaultValues().backgroundColor;
```

Comment:
this seems wrong. it is static.
```suggestion
      canvas.backgroundColor = canvas.constructor.getDefaultValues().backgroundColor;
```

Why not expose `restoreDefaultValues` or whatever?

Reply:
because we are not offering any functionality on default values, as we weren't before.
There was just a clean up of the export from defaults, and a chance to configure them.
And i don't think is useful to expand in that direction.